src/sst/elements/ariel/arielcore.h

```diff
@@ -52,6 +52,7 @@
 
 #include "ariel_shmem.h"
 #include "arieltracegen.h"
+#include <../Opal/Opal_Event.h>
```

Comment:
Should be "sst/elements/Opal/Opal_Event.h"

Reply:
@gvoskuilen Should I close this PR, change the code and create a new PR?

Reply:
@VamseeReddyK no need to close the PR, just make the change and push it to your fork